modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PageMemoryLazyAllocationWithPDSTest.java

```diff
@@ -64,21 +64,15 @@ public void testClientNodeRestart() throws Exception {
 
         IgniteEx srv = startSrv()[0];
 
-        client = true;
-
-        IgniteEx clnt = startGrid(2);
+        IgniteEx clnt = startClientGrid(2);
 
         createCacheAndPut(clnt);
 
         stopAllGrids(false);
 
-        client = false;
-
         srv = startSrv()[0];
 
-        client = true;
-
-        IgniteCache<Integer, String> cache = startGrid(2).cache("my-cache-2");
+        IgniteCache<Integer, String> cache = startClientGrid(2).cache("my-cache-2");
```

Comment:
Server node starts here, because client mode is set by `client` field of superclass.

Reply:
Fixed. Thanks.